mdtraj/core/trajectory.py

```diff
@@ -1981,10 +1981,7 @@ def make_molecules_whole(self, inplace=False, sorted_bonds=None):
         if inplace:
             result = self
         else:
-            result = Trajectory(xyz=self.xyz, topology=self.topology,
-                                time=self.time,
-                                unitcell_lengths=self.unitcell_lengths,
-                                unitcell_angles=self.unitcell_angles)
+            result = self[:]
```

Comment:
Are these changes the same as in [1812](https://github.com/mdtraj/mdtraj/pull/1812/files#diff-1b104b5984a85d8b849914ff10c636e498d9b75091dc845b9595a740f6580503R1984)?

Reply:
Ah yes they are - apologies for the duplication.